lib/FilesHooks.php

```diff
@@ -283,6 +286,9 @@ public function fileMove($oldPath, $newPath) {
 		}
 
 		$this->oldAccessList = $oldAccessList;
+
+		// now its save to set moveCase
+		$this->moveCase = $moveCase;
```

Comment:
I still don't get why this is safer to set it here

Reply:
There are 2 Signals invoked: 
https://github.com/nextcloud/activity/blob/06aabffe31252b54caf1fd439f65e48cf1971b95/lib/AppInfo/Application.php#L166-L167

The function `fileMove` is called before the action and the function `fileMovePost` afterwards.

`$this->moveCase ` gets only defined in function `fileMove`. 

In `fileMovePost` the `$this->moveCase` is consumed and will be reset

https://github.com/nextcloud/activity/blob/06aabffe31252b54caf1fd439f65e48cf1971b95/lib/FilesHooks.php#L310-L313

If the value of `$this->moveCase` is not set the function returns immediately.

https://github.com/nextcloud/activity/blob/06aabffe31252b54caf1fd439f65e48cf1971b95/lib/FilesHooks.php#L295-L299

Conclusion:
@artonge It is save the set the variable at the end cause everything **is** calculated  and there was no Error/Exception.
-> Ready to go to next stage

Currently it goes into post process and the `$this->oldAccessList` Variable is not defined because it gets defined at the end and in my case there was an exception thrown by 3rd party (circle) during calculate accesslist. :cry: 

https://github.com/nextcloud/activity/blob/06aabffe31252b54caf1fd439f65e48cf1971b95/lib/FilesHooks.php#L284-L286